src/js/switchTab.js

```diff
@@ -0,0 +1,96 @@
+import AddBook from './addBook';
+import BookList from './bookList';
+import ContactUs from './contactUs';
+import selectCategory from './utils/formCategorySelector';
+import NewBookAddedSuccess from './newBookAdded';
+import getFormValues from './utils/getFormValues';
+import AddNewBook from './addNewBookApp';
+import returnLocalItem from './utils/returnLocalItem';
+
+let booksInfo = [];
+let tempBook;
+
+const setLocalStorage = booksInfo => {
+  localStorage.setItem('storeLocal', JSON.stringify(booksInfo));
+};
+
```

Comment:
This js file seems to have a lot of exported functions in it. This goes against the principle of SOLID pattern you are trying to achieve. I method should be in it's own file in a way that it can be re-usable. Or better still you can re-factor all this into a single class.